src/frontend/src/features/utils/mail-helper.tsx

```diff
@@ -41,7 +41,9 @@ class MailHelper {
      * then render HTML ready for email through react-email.
      */
     static async markdownToHtml(markdown: string) {
-        return renderToString(<Markdown>{markdown}</Markdown>);
+        return renderToString(<Markdown>{markdown}</Markdown>)
+            .replace(/(^<div data-id="react-email-markdown">|<\/div>$)/g, '')
```

Comment:
if we don't have `<div data-id="react-email-markdown">` anymore, this could remove a closing `</div>`